hooks/tk-multi-publish2/tk-maya/basic/publish_turntable.py

```diff
@@ -562,13 +573,60 @@ def set_ui_settings(self, widget, settings):
             unreal_versions,
             cur_settings["Unreal Engine Version"],
         )
+        widget.unreal_setup_widget.unreal_engine_widget.set_path(
+            cur_settings["Unreal Engine Path"]
+        )
         widget.unreal_setup_widget.set_unreal_project_path_template(
             cur_settings["Unreal Project Path Template"]
         )
+        widget.unreal_setup_widget.unreal_project_widget.set_path(
+            cur_settings["Unreal Project Path"]
+        )
         widget.unreal_turntable_map_widget.setText(cur_settings["Turntable Map Path"])
         widget.unreal_sequence_widget.setText(cur_settings["Sequence Path"])
         widget.unreal_turntable_asset_widget.setText(cur_settings["Turntable Assets Path"])
 
+    def load_saved_ui_settings(self, settings):
+        """
+        Load saved settings and update the given settings dictionary with them.
+
+        :param settings: A dictionary where keys are settings names and
+                         values Settings instances.
+        """
+        # Retrieve SG utils framework settings module and instantiate a manager
+        fw = self.load_framework("tk-framework-shotgunutils_v5.x.x")
```

Comment:
Is possible to manage frameworks instances as an attribute of the class that is created once?

Reply:
I don't think there is any guarantee you'll get the same instance of the TK Hook. If you want to persist values, you need to store them in properties.

Reply:
Thanks